### PR TITLE
ui: editor information button

### LINKED ISSUE #120: ui: editor information button
Add a user information button

### PR DESCRIPTION
**Pull request by mayramtv**
Resolves #120 

Created a information button on the side of the Delete button. This button will display information when hovered. I added this changes to the Editor.vue

I added some points for user information, feel free to add more information. r let me know and I can make any changes.

@lenabaidakova Could you please review this info button.
Thank you.

### CODE REVIEW

File: components/Editor.vue [hunk 1]
```diff
@@ -186,6 +187,22 @@ const getJournalUpdatedDate = () => {
         >
           Delete
         </UButton>
+        <!-- Information button -->
+        <div
```
Comment (lenabaidakova) [inline components/Editor.vue:191]: @mayramtv , looks cool :) 
Just one suggestion: let's use [Tooltip](https://ui.nuxtlabs.com/overlays/tooltip) or [Popover](https://ui.nuxtlabs.com/overlays/popover) component (popover can be available on [hover](https://ui.nuxtlabs.com/overlays/popover#mode) ).
In this case, information text will overlay content and editor or Delete button won't jump

Comment (mayramtv) [reply components/Editor.vue:191]: @lenabaidakova Thank you for the tip. That was what I was trying to aim for, but I didn't know how. I'll change it today.

Comment (mayramtv) [reply components/Editor.vue]: @lenabaidakova I updated the button with the popover. Thank you for reviewing the PR

**Review by lenabaidakova — APPROVED** [review]
Looks great, thank you ☺️

**Pull request merged**